app/Libraries/Search/BeatmapsetSearchRequestParams.php

```diff
@@ -227,6 +227,7 @@ private function parseQuery(): void
             'dr' => 'drain',
             'keys' => 'keys',
             'length' => 'hitLength',
+            'od' => 'od',
```

Comment:
similarly here should map to match the attribute in transformer (although looking again, the normalization should've been done in query parser instead)